samples/crypto/tls/Kconfig

```diff
@@ -0,0 +1,38 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+source "Kconfig.zephyr"
+
+menu "PSA TLS sample"
+
+menuconfig TEST_RSA_CERTIFICATE_ENABLED
+	bool "Use RSA certificate in PSA TLS sample"
+	default n
+	help
+	  Enable RSA certificates when testing ciphers that supports RSA for authentication.
+	  If set to false, ECDSA certifcates will be used.
+
+menuconfig TEST_TLS_SERVER
```

Comment:
Could the server/client configs be implemented as a `choice`?

Reply:
Good point. I will look into this.